content/blog/2021-12-17-react-conf-2021-recap.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "React Conf 2021 Recap"
+author: [jtannady, rickhanlonii]
```

Comment:
author: [[jtannady](https://github.com/jtannady), [rickhanlonii](https://github.com/rickhanlonii)]

Reply:
Blog authors are already in the correct format.

Reply:
aw, i don't saw it